posts/2019/11/Files-Improvements.md

```diff
@@ -0,0 +1,77 @@
+<!--
+.. title: File management improvements in Spyder4
+.. slug: File-management-improvements-in-Spyder4
+.. date: 2019-11-01 12:00:00 UTC-05:00
+.. author: Juanita Gomez & Gonzalo Peña
+.. tags: Labs, Spyder
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+
+Spyder 4.0 is almost ready! It has been in the making for 1.5 years, and it contains lots of interesting new features. In this post we will focus on the files plugin where several improvements have been made regarding file management and interface. 
+
+## Simplified interface
+
+In order to simplify the file explorer interface, by default now the columns corresponding to size and kind are hidden. This feature can be activated or deactivated by either accessing the Pane Menu or by right clicking the header directly. In both, cases you can select which columns to display in the file explorer.
+
+![Pane Menu](/images/spyder-files-pane-menu.png)
+<!-- TEASER_END -->
+
+![Header Menu](/images/spyder-files-header-menu.png)
+
+## Custom file associations
+
+First, we added the possibility to associate different external applications to open specific file extensions. By going to the “Files” option in the preferences window, it is possible to set the file associations by, selecting the File Types and the Associated applications that should be used to open this specific file extensions. 
+
+![File associations](/images/spyder-files-file-associations.png)
+
+Once this configuration is set you can go to the files pane in Spyder and whenever you open a file with the selected extension, it will open automatically with the selected associated application. Additionally, when you right click a file with one of this extensions, you will find an “open with” option that will show the application associated for this extension.
+
+![Open With](/images/spyder-files-open-with.png)
+
+## Single click open on file explorer
```

Comment:
```suggestion
## Single click open option
```

* Use correct UI name
* Remove redundant mention of Files pane for conciseness and consistency with the other headings